### PR TITLE
New package: ZenBrowser.Zen version 1.0.0-a.26

### PR DESCRIPTION
**Pull request by ptsouchlos**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/168542)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

Hi @DeveloperPaul123,

It seems like installer switches might be needed for this package.

**Comment by DeveloperPaul123** [comment]
Hmm I thought I had specified the `/S` flag for this package. I'll take a look. Thanks

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 887.)

File: manifests/z/ZenBrowser/Zen/1.0.0-a.26/ZenBrowser.Zen.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ZenBrowser.Zen
+PackageVersion: 1.0.0-a.26
+InstallerType: exe
+UpgradeBehavior: install
+ReleaseDate: 2024-08-21
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/z/ZenBrowser/Zen/1.0.0-a.26/ZenBrowser.Zen.installer.yaml:10]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
From the error message, I'm guessing this is a 64-bit application. Our `x86` validations are performed on 32-bit OSes.

> Automatic Validation ended with: 
Edit: image

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**